concrete/config/concrete.php

```diff
@@ -603,6 +603,11 @@
             'get_available_updates' => 'http://www.concrete5.org/tools/update_core',
             'inspect_update' => 'http://www.concrete5.org/tools/inspect_update',
         ],
+        // Set to true to skip checking if there's a newer core version available (useful for example if the core is upgraded via composer) 
+        'skip_core' => false,
+        // Space-separated list of package handles that shouldn't be checked for new versions in marketplace (useful for example if the core is upgraded via composer)
+        // Set to true to skip all the packages
+        'skip_packages' => '',
```

Comment:
Pretty weird this can be either a boolean or a comma separated list (string). Should you just have 1 config setting for an array with strings in it OR have this skip packages as another config setting... Seems to be not error-prone to me.

Reply:
Having one config key is IMHO much simpler to be maintained.
For example, with this approach, you only have to set the `concrete.updates.skip_packages` to:
- `true` to disable all packages
- a space-separated list of package handles to only disable a selected list of packages.

If we'd have two settings (eg `concrete.updates.skip_packages.all` and `concrete.updates.skip_packages.list`), what should the core do in the following case?
- `concrete.updates.skip_packages.all` is set to `true`
- and `concrete.updates.skip_packages.list` is set to a list of package handles
The core should take care of the precedence, and you (the user/installation manager) have to be aware of the precedence... this is quite misleading and unnecessarily complex IMHO...